articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,64 @@
+
+### Quantum measurement
+
+Now, let’s say your friend comes over and wants to take a picture of you exercising. Most likely, they’ll get a blurry image of you turning somewhere between all the way left and all the way right.
+
+But if you’re a quantum particle, an interesting thing happens. No matter where you are when they take the picture, it will always show you either all the way left or all the way right – nothing in-between.
+
+This is because the act of observing or measuring a quantum particle **collapses** the superposition state (also known as **decoherence**) and the particle takes on a classical binary state of either 1 or 0.
+
+This binary state is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever (just like your picture) and will always be a 1 or 0, and cannot be used for more calculations. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be useful again.
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle correlates to the other particles as well, almost instantaneously.
+
+In addition to this interdependency, particles can maintain this connection even when separated over incredibly large distances, even light-years. The effects of quantum measurement also apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. Because there is a correlation between the entangled qubits, measuring the state of one qubit provides information about the state of the other qubit – this particular property is very helpful in quantum computing. These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and can play a role in some cryptography processes.
+
+### Qubits and probability
+
+Classical computers store and process information in bits, which can have a state of either 1 or 0, but never both. The equivalent in quantum computing is the **qubit**, which represents the state of a quantum particle. Because of superposition, qubits can either be 1 or 0, or both, or anything in between. Depending on its configuration, a qubit has a certain *probability* of collapsing to 1 or 0, and this probabilistic state is where the power of quantum computing excels.
+
+In classical computing, uncertainty is not a good thing. In quantum computing, uncertainty is built-in.
```

Comment:
Monte Carlo methods can be very useful classically as well. It may be helpful to note that quantum computing isn't just classical computing with randomness.